scripts/cli.dart

```diff
@@ -0,0 +1,215 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
I wonder if this should live in pub (package:flutter_tools?) so we can re-use it across other plugin repos (e.g. FlutterFire), and other plugin authors can depend on it as well. I don't have strong feelings about it though. Probably some who hacks on flutter_tools more should review.